### PR TITLE
Kotlin 1.4.21

### PR DESCRIPTION
**Pull request by antonioseric**

### CODE REVIEW

**Review by moljac — APPROVED** [review]

**Review by moljac — APPROVED** [review]

**Comment by 4brunu** [comment]
Any news on this? 🙂

File: Android/Kotlin/config.json [hunk 1]
```diff
@@ -72,21 +72,14 @@
...
 			"version" : "1.3.4",
 			"nugetId" : "Xamarin.KotlinX.Coroutines.CoroutinesAndroid",
 			"nugetVersion" : "1.3.4"
-		},
-		{
-			"groupId" : "org.jetbrains.kotlinx",
-			"artifactId" : "kotlinx-coroutines-core",
-			"version" : "1.3.4",
-			"nugetId" : "Xamarin.KotlinX.Coroutines.Core",
-			"nugetVersion" : "1.3.4"
```
Comment (mattleibow) [inline Android/Kotlin/config.json:89]: Is this no longer needed?

File: Android/Kotlin/samples/KotlinSampleLibrary/KotlinSampleLibrary.csproj [hunk 2]
```diff
@@ -42,7 +42,7 @@
     <Reference Include="Mono.Android" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Kotlin.StdLib" Version="1.4.20" />
+    <PackageReference Include="Xamarin.Kotlin.StdLib.Jdk7" Version="1.4.21" />
```
Comment (mattleibow) [inline Android/Kotlin/samples/KotlinSampleLibrary/KotlinSampleLibrary.csproj:45]: Why was this change needed? Do we no longer need the stdlib?

**Comment by 4brunu** [comment]
Since azure pipelines run with success, could this PR be merged and published in nuget? Thanks 😊

File: Android/Kotlin/config.json (hunk 1 above)
Comment (antonioseric) [reply Android/Kotlin/config.json:89]: we should merge #1050 first maybe

File: Android/Kotlin/samples/KotlinSampleLibrary/KotlinSampleLibrary.csproj (hunk 2 above)
Comment (antonioseric) [reply Android/Kotlin/samples/KotlinSampleLibrary/KotlinSampleLibrary.csproj:45]: `Xamarin.Kotlin.StdLib.Jdk7` has dependency to `Xamarin.Kotlin.StdLib`

**Comment by 4brunu** [comment]
Any news on this?
@mattleibow @moljac @jpobst

**Comment by moljac** [comment]
merged and published from other PR
